runtime/doc/lua.txt

```diff
@@ -2392,4 +2392,35 @@ trust({opts})                                             *vim.secure.trust()*
         • true and full path of target file if operation was successful
         • false and error message on failure
 
+
+==============================================================================
+Lua module: inspector                                          *lua-inspector*
+
+get({bufnr}, {row}, {col}, {filter})                     *vim.inspector.get()*
+    Get all the items at a given buffer position
+
+    Parameters: ~
+      • {bufnr}   buffer|nil defaults to the current buffer
+      • {row}     (number|nil) 0-indexed row. Defaults to the current cursor
+      • {col}     (number|nil) 0-indexed col. Defaults to the current cursor
+      • {filter}  (table|nil) a table with key-value pairs to filter the items
+                  • syntax (boolean): include syntax based highlight groups
+                    (defaults to true)
+                  • treesitter (boolean): include treesitter based highlight
+                    groups (defaults to true)
+                  • extmarks (boolean): include extmarks (defaults to true)
+                  • extmarks_with_hl_group (boolean): when false, also include
+                    extmarks without a hl_group (defaults to true)
+                  • semantic_tokens (boolean): include semantic tokens
+                    (defaults to true)
+
+show({bufnr}, {row}, {col}, {filter})                   *vim.inspector.show()*
+    Show all the items at a given buffer position
```

Comment:
Since this is a "show all the things" interface, it can live directly on vim.show_pos(). Don't need a new "inspector" module.